keepassxc-browser/background/event.js

```diff
@@ -206,7 +206,7 @@ kpxcEvent.onCheckUpdateKeePassXC = function(callback, tab) {
 };
 
 kpxcEvent.onUpdateAvailableKeePassXC = function(callback, tab) {
-    callback(keepass.keePassXCUpdateAvailable());
+    callback(page.settings.checkUpdateKeePassXC > 0 ? keepass.keePassXCUpdateAvailable() : false);
```

Comment:
I think it would be better to just never enter this function to begin with.

Reply:
That's not a function. It's the variable where the setting is stored. `0` means it never checks for the update. The real function where the check happens is `keepass.keePassXCUpdateAvailable()`.

Reply:
No what I meant was this whole block is an anonymous function, just don't call it (ie, ```kpxcEvent.onUpdateAvailableKeePassXC()```) if the update check is disabled.

Reply:
Nevermind, I see how this is used now, I was confused.